2021-12-03 03:03:34,941 INFO    MainThread:55144 [wandb_setup.py:_flush():71] setting env: {}
2021-12-03 03:03:34,941 INFO    MainThread:55144 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-03 03:03:34,942 INFO    MainThread:55144 [wandb_init.py:_log_setup():371] Logging user logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_030334-1pnl4ps5/logs/debug.log
2021-12-03 03:03:34,942 INFO    MainThread:55144 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211203_030334-1pnl4ps5/logs/debug-internal.log
2021-12-03 03:03:34,942 INFO    MainThread:55144 [wandb_init.py:init():404] calling init triggers
2021-12-03 03:03:34,942 INFO    MainThread:55144 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'max_epochs': 150, 'batch_size': 100}
2021-12-03 03:03:34,942 INFO    MainThread:55144 [wandb_init.py:init():449] starting backend
2021-12-03 03:03:34,943 INFO    MainThread:55144 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-03 03:03:34,978 INFO    MainThread:55144 [backend.py:ensure_launched():199] starting backend process...
2021-12-03 03:03:35,044 INFO    MainThread:55144 [backend.py:ensure_launched():205] started backend process with pid: 55172
2021-12-03 03:03:35,086 INFO    MainThread:55144 [wandb_init.py:init():458] backend started and connected
2021-12-03 03:03:35,098 INFO    MainThread:55144 [wandb_init.py:init():519] updated telemetry
2021-12-03 03:03:35,169 INFO    MainThread:55144 [wandb_init.py:init():549] communicating current version
2021-12-03 03:03:37,590 INFO    MainThread:55144 [wandb_init.py:init():554] got version response 
2021-12-03 03:03:37,590 INFO    MainThread:55144 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-03 03:03:37,900 INFO    MainThread:55144 [wandb_init.py:init():592] starting run threads in backend
2021-12-03 03:03:42,908 INFO    MainThread:55144 [wandb_run.py:_console_start():1816] atexit reg
2021-12-03 03:03:42,910 INFO    MainThread:55144 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-03 03:03:42,911 INFO    MainThread:55144 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-03 03:03:42,913 INFO    MainThread:55144 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-03 03:03:42,913 INFO    MainThread:55144 [wandb_init.py:init():619] run started, returning control to user process
2021-12-03 03:03:42,915 INFO    MainThread:55144 [wandb_watch.py:watch():43] Watching
2021-12-03 03:04:34,496 INFO    MainThread:55144 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-03 03:04:34,509 INFO    MainThread:55144 [wandb_run.py:_restore():1758] restore
2021-12-03 03:04:36,964 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 03:04:37,350 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-03 03:04:37,523 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7033
}

2021-12-03 03:04:37,627 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7033
}

2021-12-03 03:04:37,735 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7033
}

2021-12-03 03:04:37,843 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7033
}

2021-12-03 03:04:37,946 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:38,051 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:38,154 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:38,260 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:38,364 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:38,980 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}

2021-12-03 03:04:39,516 INFO    MainThread:55144 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7033
  total_bytes: 7033
}
local_info {
}

2021-12-03 03:04:42,669 INFO    MainThread:55144 [wandb_run.py:_append_files():2184] logging synced files
